docker/README.md

```diff
@@ -80,6 +80,13 @@ all the required artifacts:
 ./build.sh -r 4
 ```
 
+Set the environment variable `TRINO_GATEWAY_BASE_IMAGE` to use a specific base image to build Trino Gateway image.
+
+```bash
+export TRINO_GATEWAY_BASE_IMAGE=<image>
+./build.sh
```

Comment:
build.sh already offers some arguments h,a,r,j to changer the behavior. Why do we need to use an environment variable for the base image?

Reply:
do we want to use these arguments to change the base image then? Because right now the base image is using the one on dockerhub, but we will need to use our own base image to build the docker image.

Reply:
The script also uses a bunch of env variables.. so its kinda inconsistent .. I think of the trino image more like the java home path to be honest